shotover-proxy/Cargo.toml

```diff
@@ -8,6 +8,10 @@ license = "Apache-2.0"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+[features]
+# Include the WIP Protect transform in the public API
+protect = []
```

Comment:
I'm thinking this should be called `int-test` or similar so it's clear it has no purpose other than integration testing. Unless it's not just for integration testing?

Reply:
In theory it would allow users who are willing to compile from source to try out new unreleased features.
Maybe we should call the feature `alpha-transforms` to allow for that case while also avoiding having lots of different features for each alpha transform floating around.

> Why are we not including the Protect transform in the public API?

New alpha transforms should not be public because:
* We dont want non-functional features to be available to users.
* We dont want to commit to a stable API yet.

Reply:
That makes sense to me. I agree with the name suggestion `alpha-transforms`.